targets/TARGET_STMBLUE/us_ticker_api.c

```diff
@@ -0,0 +1,137 @@
+/* mbed Microcontroller Library
+ * @Author: Antonio O.
+ *
+ * NOTE:
```

Comment:
please add license header file (also with SPDX identifier). Look at files in platform/drivers folders for instance